web/packages/teleterm/src/ui/uri.ts

```diff
@@ -23,6 +23,8 @@ export const paths = {
   leafCluster: '/clusters/:rootClusterId/leaves/:leafClusterId',
   server:
     '/clusters/:rootClusterId/(leaves)?/:leafClusterId?/servers/:serverId',
+  serverLeaf:
```

Comment:
should we keep the name consistent with leaf first? `leafServer`? I see instances of `leafServer` in the codebase but no `serverLeaf`

Reply:
I think I just wanted the key to start with the most important descriptor in this set of keys (what kind of resource this path is for) and only then followed by a minor descriptor of whether it's for a leaf resource or not. So it's referring less to the concept of a leaf server and more to the concept of a path for servers that are leaf servers.

Alas, we already have `rootCluster` and `leafCluster` within the same object, I guess I could rename then to `clusterRoot` and `clusterLeaf` if I wanted to be consistent. 🤔

Reply:
I see. I really like that idea of not describing the "item" but describing the path and it makes _much_ more sense as a path descriptor to follow your `serverLeaf` convention. feels very REST-y. I'm ok with keeping it then and, if you feel compelled to change the others thats fine too but, either way, I see the light now. Thanks for the insight